cmd/root.go

```diff
@@ -51,7 +53,7 @@ const (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "./scorecard --repo=<repo_url> [--checks=check1,...] [--show-details]",
+	Use:   "./scorecard --repo=<repo_url> [--checks=check1,...] [--show-details] or ./scorecard --npm=<npm packagename> [--checks=check1,...] [--show-details]",
```

Comment:
need to think of a better way to phrase this, this is becoming too long.

Reply:
I agree